docs:#804update code snippets for starting seed nodes (#805)
redpanda-data/documentation: modules/deploy/pages/deployment-option/cloud/managed-connectors/create-snowflake-connector.adoc

Refer to the https://docs.snowflake.com/en/user-guide/kafka-connector-overview#kafka-connector-limitations[Snowflake Kafka Connector Limitations^]
documentation for details.

> Snowflake tables contain metadata with offset information. If a Redpanda topic is dropped and recreated after the initial sync to Snowflake it can cause issues due to mismatch of offsets, which can result in new data not being sent to Snowflake. When deleting and/or recreating Redpanda topics that have already sent data to Snowflake using the connector, you must delete the corresponding Snowflake table as well.

Reviewer: ```suggestion
Snowflake tables contain metadata with offset information. If a Redpanda topic is dropped and recreated after the initial sync to Snowflake, it can cause issues due to a mismatch of offsets. This can result in new data not being sent to Snowflake. If you delete or recreate Redpanda topics that have already sent data to Snowflake using this connector, you must delete the corresponding Snowflake table as well.
```

Reviewer 2: the suggestion now reads like the delete/recreate of topic is done via the connector which is not the case.

Reviewer: I see what you mean, does this work now?

> Snowflake tables have metadata containing offset information. If a Redpanda topic has been dropped and recreated since the initial sync to Snowflake it can cause issues due to mismatch of offsets which can mean new data is not sent to Snowflake. In the case of deleting/recreating Redpanda topics that have already been sent data to Snowflake via the connector, you must delete the corresponding Snowflake table as well.

Reviewer 3: ```suggestion
Snowflake tables contain metadata with offset information. If a Redpanda topic is dropped and recreated after the initial sync to Snowflake it can cause issues due to mismatch of offsets, which can result in new data not being sent to Snowflake. When deleting and/or recreating Redpanda topics that have already sent data to Snowflake using the connector, you must delete the corresponding Snowflake table as well.
```

|*Config:value.converter has provided value:com.snowflake.kafka.connector.records.SnowflakeJsonConverter. If ingestionMethod is:snowpipe_streaming, Snowflake Custom Converters are not allowed.*
|Use `STRING` for the `Kafka message value format`.

> |* Ignoring invalid task provided offset mytopic-12/OffsetAndMetadata{offset=7182, leaderEpoch=null, metadata=''} -- not yet consumed, taskOffset=7182 currentOffset=6804"}*

Reviewer: ```suggestion
|*Ignoring invalid task provided offset mytopic-12/OffsetAndMetadata{offset=7182, leaderEpoch=null, metadata=''} -- not yet consumed, taskOffset=7182 currentOffset=6804"}*
```

> |Snowflake tables store metadata containing offset info. Likely this topic has been dropped and recreated since the initial sync to Snowflake causing issue with mismatch of offsets. When deleting/recreating Redpanda topics that have already been sent data to Snowflake,you need to delete the corresponding Snowflake table as well.

Reviewer 3: ```suggestion
|Snowflake tables store metadata containing offset information. It is likely that this topic has been dropped and recreated since the initial sync to Snowflake, and this may cause a mismatch of offsets. When deleting and/or recreating Redpanda topics that have already sent data to Snowflake, you must delete the corresponding Snowflake table as well.
```

> |Snowflake tables store metadata containing offset information. It is likely that this topic has been dropped and recreated since the initial sync to Snowflake, and this may cause a mismatch of offsets. When deleting and/or recreating Redpanda topics that have already sent data to Snowflake, you must delete the corresponding Snowflake table as well.

Reviewer: ```suggestion
|Snowflake tables store metadata that contains offset information. It is likely that this topic was dropped and recreated since the initial sync to Snowflake, which caused a mismatch of offsets. When deleting or recreating Redpanda topics that have already sent data to Snowflake, you must delete the corresponding Snowflake table as well.
```

Reviewer 2: Hi @asimms41  do I need to do [commit suggestion] on each of these?

Reviewer: Hi @jason-da-redpanda, That would be great, or I can, if you are happy with them.
